make_prg/seq_utils.py

```diff
@@ -54,14 +54,23 @@ def ungap(seq: str) -> str:
     return seq.replace("-", "")
 
 
-def get_interval_seqs(interval_alignment: MSA):
+def count(iterable) -> int:
+    return sum(1 for elem in iterable)
+
+
+def get_alignment_seqs(alignment: MSA):
```

Comment:
```suggestion
def get_alignment_seqs(alignment: MSA) -> Generator[str]:
```
will need to import `Generator` from `typing` too

Reply:
Agreed and done